src/routes/PastTrips/PastTripsCard.tsx

```diff
@@ -1,13 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import SVGIcon from '../../components/SVGIcon';
+
+const PastTripsCard = ({ trips, deleteTrip }: any) => {
+  const [isDeleted, setIsDeleted] = useState(false);
```

Comment:
This name sounds like if the component `isDeleted` it's already gone. But is more like a previous state to being actually deleted.

we could call this `isBeingDeleted` to be super explicit haha. But I'm open to other suggestions!

Reply:
Make sense, yes, I name variables in the moment, I need to fix it later